app/assets/stylesheets/_entries.scss

```diff
@@ -1,48 +1,61 @@
 .new_entry_wrapper {
   background: #fff;
   box-shadow: 0px 2px 2px rgba(83, 63, 63, 0.3);
-  @include pad(10% 3%);
+  padding-bottom: 20px;
 
-  @include media($medium-screen) {
-    @include pad(5%)
+  .button {
+    width: 100%;
   }
 
-  #new_entry, .edit_entry {
+  h2 {
+    padding: 25px;
+  }
 
-    .button {
-      width: 100%;
+  .input {
+    margin: 10px 0;
+    background-color:white;
+    input {
+      -webkit-font-smoothing: antialiased;
+      font-smoothing: antialiased;
+      resize: none;
     }
+  }
 
-    .input {
-      margin: 10px 0;
+  .selectize-dropdown,
+  .selectize-input,
+  .selectize-input input {
+    font-family: "Open Sans";
+    font-size: $base-font-size;
+    background-color:white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space